### PR TITLE
Fix handling events without names in activities

### PR DESCRIPTION
**Pull request by nickvergessen**
Before | After
---|---
Bildschirmfoto vom 2022-05-24 17-05-50 | Bildschirmfoto vom 2022-05-24 17-05-35

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
Magic

File: apps/dav/lib/CalDAV/Activity/Provider/Event.php [hunk 1]
```diff
@@ -88,9 +88,9 @@ protected function generateObjectParameter(array $eventData) {
 		$params = [
 			'type' => 'calendar-event',
 			'id' => $eventData['id'],
-			'name' => $eventData['name'],
-
+			'name' => $eventData['name'] !== '' ? $eventData['name'] : $this->l->t('Untitled event'),
```
Comment (tcitworld) [inline apps/dav/lib/CalDAV/Activity/Provider/Event.php:91]: Maybe `trim` the name too?

**Review by tcitworld — APPROVED** [review]

File: apps/dav/lib/CalDAV/Activity/Provider/Event.php (hunk 1 above)
Comment (tcitworld) [inline apps/dav/lib/CalDAV/Activity/Provider/Event.php:91]: The `l` property only gets initialized in `parse` for some reason, is it absolutely safe to use here?

Comment (nickvergessen) [reply apps/dav/lib/CalDAV/Activity/Provider/Event.php]: done

**Pull request merged**